shared/src/ledger/mod.rs

```diff
@@ -1,8 +1,11 @@
 //! The ledger modules
 
 pub mod gas;
+#[cfg(feature = "wasm-runtime")]
```

Comment:
If we move all the ibc impl for `native_vp::Ctx` into the `ibc` module (https://github.com/anomanetwork/anoma/pull/329/files#r665105370), we can then only put the condition on the `ibc` module and in `native_vp` on the ibc module import for genesis initialization and `Ibc::init_genesis_storage`